### ISSUE TITLE
About KNN evaluation and finetune settings

### ISSUE THREAD

**Issue by SLime1998** [Levigty/AimCLR#5]
Hi, Thank you for sharing your work!  You used KNN evaluation method in your paper, but I did not find the related implementation details in the article, nor did I find the corresponding code. Would please provide some more details of the KNN evaluation? And as to the finetune setting, did you adjust the weight decay when doing finetune? Looking forward to your reply

**Comment by Levigty** [comment]
Hi. For the KNN evaluation, we use the [knn_monitor](https://github.com/PatrickHua/SimSiam/blob/01d7e7811ac7b864bf8adccc8005208878208994/tools/knn_monitor.py) like other contrastive learning methods. For finetune setting, we set weight decay to 1e-4, learning rate 0.01 or 0.001.

**Comment by SLime1998** [comment]
Thanks a lot for your help!
